ui/src/app/components/pods/pods.scss

```diff
@@ -15,7 +15,6 @@ $POD_SIZE: 30px;
     }
     &__container {
         margin-top: 10px;
-        // border: 1px solid $argo-color-gray-5;
```

Comment:
Do you want to remove commented code?

Reply:
Yeah, this was from old styles that I decided to remove